ckan/migration/versions/082_add_metadata_created.py

```diff
@@ -0,0 +1,15 @@
+def upgrade(migrate_engine):
+    migrate_engine.execute('''
+        ALTER TABLE package_revision
+            ADD COLUMN metadata_created timestamp without time zone;
+        ALTER TABLE package
+            ADD COLUMN metadata_created timestamp without time zone
+            DEFAULT now();
```

Comment:
Let these be null in the ALTER TABLE then postgres won't have to walk the whole package table on this statement only to overwrite it in the update below.